src/wallet/typings/Wallet.d.ts

```diff
@@ -0,0 +1,62 @@
+export interface WalletFile {
```

Comment:
Might be consistent to rename this to `WalletLike`

Reply:
updated to WalletLike